src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpointUnitTests.java

```diff
@@ -256,6 +256,45 @@ public void findCollectionExerciseNotFound() throws Exception {
         .andExpect(handler().methodName("getCollectionExercise"));
   }
 
+  /**
+   * Test to get collection exercise with get events throws CTP Exception.
+   *
+   * @throws CTPException exception thrown
+   */
+  @Test
+  public void
+      findCollectionExerciseGetEventsThrowsCTPExceptionSuccessfullyRetrievesCollectionExercise()
+          throws Exception {
+    when(collectionExerciseService.findCollectionExercise(COLLECTIONEXERCISE_ID1))
+        .thenReturn(collectionExerciseResults.get(0));
+    when(collectionExerciseService.getCaseTypesList(collectionExerciseResults.get(0)))
+        .thenReturn(caseTypeDefaultResults);
+    when(surveyService.findSurvey(UUID.fromString("31ec898e-f370-429a-bca4-eab1045aff4e")))
+        .thenReturn(surveyDtoResults.get(0));
+    when(eventService.getEvents(COLLECTIONEXERCISE_ID1)).thenThrow(CTPException.class);
```

Comment:
Complains about`unchecked generics array creation for varargs param` so to keep it consistent with what it is currently like throughout the file try:
`when(eventService.getEvents(COLLECTIONEXERCISE_ID1)).thenThrow(new CTPException(CTPException.Fault.SYSTEM_ERROR));`